docs/guide/data_migration.md

```diff
@@ -0,0 +1,160 @@
+- `delete` - deletes an existing item
+
+If you do not provide the `mode` parameter, the command will ask you to select the mode.
+
+The following combinations of types are modes are available:
+
+||`create`|`update`|`delete`|
+|---|:---:|:---:|:---:|
+|`content`|+|+|+|
+|`content_type`|+|+||
+|`role`|+|+|+|
+|`content_type_group`|+|+||
+|`user`|+|+||
+|`user_group`|+||+|
+|`language`|+|||
+|`object_state_group`|+|||
+|`object_state`|+|||
+|`section`|+|+||
+|`location`||+||
+
+### match-property
+
+The `match-property` parameter, together with `value`, enables you to select which data from the Repository to export.
+`match-property` defines what property should be used as a criterion for selecting data.
+The following properties are available (per type):
+
+- `content`
+    - `content_id`
+    - `content_type_id`
+    - `content_type_group_id`
+    - `content_type_identifier`
+    - `content_remote_id`
+    - `location_id`
+    - `location_remote_id`
+    - `parent_location_id`
+    - `user_id`
+- `content_type`
+    - `content_type_identifier`
+- `content_type_group`
+    - `content_type_group_id`
+    - `content_type_group_identifier`
+- `language`
+    - `language_code`
+- `location`
+    - `location_remote_id`
+    - `location_id`
+- `object_state`
+    - `object_state_id`
+    - `object_state_identifier`
+- `object_state_group`
+    - `object_state_group_id`
+    - `object_state_group_identifier`
+- `role`
+    - `identifier`
+    - `id`
+- `section`
+    - `section_id`
+    - `section_identifier`
+- `user`
+    - `login`
+    - `email`
+
+### value
+
+The `value` parameter, together with `match-property`, filters the Repository content that the command exports.
+`value` defines which values of the `match-property` should be included in the export.
+
+For example, to export only Article Content items, use the `content_type_identifier` match property with `article` as the value:
+
+``` bash
+bin/console ibexa:migrations:generate --type=content --mode=create --match-property=content_type_identifier --value=article
+```
+
+### file
+
+The `file` parameter defines the name of the YAML file to export to.
+
+``` bash
+bin/console ibexa:migrations:generate --type=content --mode=create --file=my_data_export.yaml
+```
+
+### user-context
+
+The `user-context` parameter enables you to run the export command as a specified User.
+The command only exports Repository data that the selected User has access to.
+
+``` bash
+bin/console ibexa:migrations:generate --type=content --mode=create --user-context=jessica_andaya
+```
+
+## Importing data
+
+To import Repository data from a YAML file, run the `ibexa:migrations:migrate` command.
+
+The command takes the file name as parameter. The file is located by default in the `src/Migrations/Ibexa` folder.
+
+``` bash
+bin/console ibexa:migrations:migrate --file=my_data_export.yaml
+```
+
+## Converting migration files
+
+You can use the `ibexa:migrations:kaliop:convert` command to convert a file
```

Comment:
I would change the paragraph in the following way:

"If you want to convert a file in the format used by the [Kaliop migration bundle](https://github.com/kaliop-uk/ezmigrationbundle) to the current 
migration format, you can use the `ibexa:migrations:kaliop:convert` command.
The source file must use the currently supported [mode and type combinations](#mode)."

Reply:
> The source file must use the currently supported mode and type combinations."
Also, this is not exactly accurate. Source file has to use kaliop modes & types. If they differ from the format we use (for example, `type: sql` can omit `mode` altogether, which in our case is `mode: execute`) the converter will recognize this and change it accordingly in output file.